phlop/procs/runtimer.py

```
@@ -35,24 +33,22 @@ def __init__(
+            "stdout": subprocess.DEVNULL,
+            "stderr": subprocess.DEVNULL,
+        }
         if not capture_output and log_file_path:
-            ekwargs.update(
-                dict(
-                    stdout=open(f"{log_file_path}.stdout", "w"),
-                    stderr=open(f"{log_file_path}.stderr", "w"),
-                ),
-            )
+            # handles must outlive this scope: passed to Popen/run and closed there
+            stdout_file = open(f"{log_file_path}.stdout", "w")  # noqa: SIM115
```

Pylint (4.0.6)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)